### PR TITLE
emacs: add pt_BR translation

### PR DESCRIPTION
**Pull request by andrecoco**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/common/emacs.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# emacs
+
+> O editor extensível, personalizável, autodocumentável, com exibição em tempo real.
+> Veja também `emacsclient`.
+> Mais informações: <https://www.gnu.org/software/emacs>.
+
+- Iniciar o Emacs e abra um arquivo:
```
Comment (navarroaxel) [inline pages.pt_BR/common/emacs.md:7]: Please use presente indicativo:
```suggestion
- Inicia o Emacs e abra um arquivo:
```

File: pages.pt_BR/common/emacs.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+`emacs {{caminho/para/arquivo}}`
+
+- Abrir um arquivo em uma linha especificada:
+
+`emacs +{{numero_linha}} {{caminho/para/arquivo}}`
+
+- Iniciar o Emacs em modo console (sem uma janela X):
```
Comment (navarroaxel) [inline pages.pt_BR/common/emacs.md:15]:
```suggestion
- Inicia o Emacs em modo console (sem uma janela X):
```

File: pages.pt_BR/common/emacs.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+`emacs --no-window-system`
+
+- Iniciar um servidor Emacs em segundo plano (acessível através do `emacsclient`):
```
Comment (navarroaxel) [inline pages.pt_BR/common/emacs.md:19]:
```suggestion
- Inicia um servidor Emacs em segundo plano (acessível através do `emacsclient`):
```

File: pages.pt_BR/common/emacs.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+`emacs --daemon`
+
+- Parar um servidor Emacs em funcionamento e todas as suas instâncias, pedindo confirmação em arquivos não salvos:
```
Comment (navarroaxel) [inline pages.pt_BR/common/emacs.md:23]:
```suggestion
- Para um servidor Emacs em funcionamento e todas as suas instâncias, pedindo confirmação em arquivos não salvos:
```

File: pages.pt_BR/common/emacs.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+> O editor extensível, personalizável, autodocumentável, com exibição em tempo real.
+> Veja também `emacsclient`.
+> Mais informações: <https://www.gnu.org/software/emacs>.
+
+- Iniciar o Emacs e abra um arquivo:
+
+`emacs {{caminho/para/arquivo}}`
+
+- Abrir um arquivo em uma linha especificada:
```
Comment (navarroaxel) [inline pages.pt_BR/common/emacs.md:11]:
```suggestion
- Abre um arquivo em uma linha especificada:
```

File: pages.pt_BR/common/emacs.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+- Parar um servidor Emacs em funcionamento e todas as suas instâncias, pedindo confirmação em arquivos não salvos:
+
+`emacsclient --eval '(save-buffers-kill-emacs)'`
+
+- Salvar um arquivo em Emacs:
+
+`Ctrl + X, Ctrl + S`
+
+- Deixar o Emacs:
```
Comment (navarroaxel) [inline pages.pt_BR/common/emacs.md:31]:
```suggestion
- Deixa o Emacs:
```

File: pages.pt_BR/common/emacs.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+- Iniciar um servidor Emacs em segundo plano (acessível através do `emacsclient`):
+
+`emacs --daemon`
+
+- Parar um servidor Emacs em funcionamento e todas as suas instâncias, pedindo confirmação em arquivos não salvos:
+
+`emacsclient --eval '(save-buffers-kill-emacs)'`
+
+- Salvar um arquivo em Emacs:
```
Comment (navarroaxel) [inline pages.pt_BR/common/emacs.md:27]:
```suggestion
- Salva um arquivo em Emacs:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @andrecoco, welcome and thanks for your submission. I left some comments.

**Comment by andrecoco** [comment]
Thanks for the comments, i'll check the other translations i did for those mistakes

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @andrecoco

**Pull request merged**